### PR TITLE
Blog post about devopsdays Berlin 2017

### PR DESCRIPTION
**Pull request by muggahtee**

### CODE REVIEW

File: _posts/2017-11-24-devopsdays-berlin-2017.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Security
+
+Jenny Duckett from Government Digital Service (the brains behind GOV.UK) championed an Open Space to talk about how we can better involve Security in the agile way we all want to work. The view from many in the session was that security was wrongly seen as just hoops to jump through to live, rather than an integral part of software development that should be present from the outset.
+
+The concensus was that developers should have access to tools that will automatically scan their code, with further security testing during the release pipelines to the testing, staging, and production environments. Though this would undoubtedly increase the chances of catching vulnerabilities, the group decided that it was important that people understood _why_ these scans were happening and what different results meant, rather than just getting a failed build and having to seek out someone from the security team to explain.
```
Comment (mitchellrj) [inline _posts/2017-11-24-devopsdays-berlin-2017.md]: "consensus"

File: _posts/2017-11-24-devopsdays-berlin-2017.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Culture
+
+One of the Open Space sessions on the first day was all around how to successfully run a Devops guild and get all areas of the business involved with the Devops mentatlity. Lots of solutions were offered, including a take on the [Lean Coffee](http://leancoffee.org) idea, pair programming as a spectator sport, and presentations in the office to talk about who is working on what.
```
Comment (mitchellrj) [inline _posts/2017-11-24-devopsdays-berlin-2017.md]: "mentality"

File: _posts/2017-11-24-devopsdays-berlin-2017.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Jenny Duckett from GDS also did a talk that focussed on cultivating a culture of learning, through teaching people how to run workshops to share knowledge. The message being that you as an individual can begin to encourage a culture of learning. If you're interested, the slide deck from Jenny's presentation can be found [here](https://speakerdeck.com/jennyd/encouraging-a-culture-of-learning-across-your-organisation).
+
+Other Open Space discussions focussed on diversity and inclusive culture, and psychological safety. This is something that there is a big focus on at Sky Betting and Gaming [write a bit more here]
```
Comment (mitchellrj) [inline _posts/2017-11-24-devopsdays-berlin-2017.md]: write a bit more here 😛

File: _posts/2017-11-24-devopsdays-berlin-2017.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+* My Open Space suggestion to talk #chatops was selected. It was great to talk to other teams that used Slack, and learn about their integrations and operational procedures. For example, at Sky Betting and Gaming we create a new channel should an incident require it, so we have a full timeline of events automatically generated.
+
+* Schlomo Schapiro covered "hands off" in both an Open Space and a talk. The Open Space looked at exactly how automation can be safely used for all operations in [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete). The argument being whether you are truly automating CRUD if you can't delete your entire estate? It was a very interesting discussion, with many conflicting opinions and a lot of blurred lines. The conversation about "hands off" as a follow on to Devops continued in his talk the next day. The summary of this talk was "root for all, or root for none". He put forward that in a Devops environment, developers and ops staff should have the same access. Either all have root, or you embrace "hands off" operations.
```
Comment (mitchellrj) [inline _posts/2017-11-24-devopsdays-berlin-2017.md]: "hands-off" and "follow-on" perhaps?

File: _posts/2017-11-24-devopsdays-berlin-2017.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+I was able to attend the conference because of the Tech Ninja fund that Sky Betting and Gaming offer to staff. I learned a great deal, and found myself asking many questions during the conference such as "how can Core Tribe integrate this into our ways of working?", "which amazing Slack integration shall I talk about next?", and a few Paulaners, "it's 2am and I'm in Berlin. Where am I staying and how do I get there?"
```
Comment (mitchellrj) [inline _posts/2017-11-24-devopsdays-berlin-2017.md]: "after a few"

**Comment by muggahtee** [comment]
Fixed the typos pointed out by @mitchellrj :)

File: _posts/2017-11-24-devopsdays-berlin-2017.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+---
+layout:     post
+title:      devopsdays Berlin 2017
+date:       2018-01-04 15:00:00
```
Comment (WheresAlice) [inline _posts/2017-11-24-devopsdays-berlin-2017.md]: This date doesn't match the date in the filename.  We can fix this just before merging though, so no need to fix it now.

**Pull request merged**